indico/modules/events/editing/models/revisions.py

```diff
@@ -176,3 +181,14 @@ def is_submitter_revision(self):
     @property
     def is_editor_revision(self):
         return self.type.is_editor_action
+
+    @listens_for(orm.mapper, 'after_configured', once=True)
```

Comment:
This block should be on the top level and not inside the class. Technically it works, but having `_mappers_configured` as a broken method in the class feels quite ugly